tests/units/anta_tests/test_connectivity.py

```diff
@@ -277,7 +319,7 @@
                 },
             },
         ],
-        "expected": {"result": "failure", "messages": ["The following port(s) have issues: {'no_lldp_neighbor': ['Ethernet2']}"]},
+        "expected": {"result": "failure", "messages": ["No LLDP neighbor(s): Ethernet2"]},
```

Comment:
under `lldpNeighbors` if we replace Ethernet1 to Ethernet11 still unit test pass but it should fail with "Port(s) not configured: Ethernet11"

Reply:
That's because of how we run the unit tests:
```
        # Test will pass if the expected message is included in the test result message
        for message, expected in zip(test_instance.result.messages, data["expected"]["messages"]):  # NOTE: zip(strict=True) has been added in Python 3.10
            assert expected in message
```

We are discussing this internally to see if we should we more strict. It would be in a separate PR anyways.